### PR TITLE
Split Stackdriver request for more than 100 pods.

### PR DESCRIPTION
**Pull request by kawych**

### CODE REVIEW

**Comment by kawych** [comment]
@piosz 
Please feel free to reassign. I'm not sure if this PR requires your attention.

File: custom-metrics-stackdriver-adapter/pkg/adapter/provider/provider.go [hunk 1]
```diff
@@ -144,3 +151,10 @@ func (p *StackdriverProvider) ListAllMetrics() []provider.MetricInfo {
 	}
 	return p.translator.GetMetricsFromSDDescriptorsResp(response)
 }
+
+func min(a, b int) int {
```
Comment (piosz) [inline custom-metrics-stackdriver-adapter/pkg/adapter/provider/provider.go]: c'mon, no min function in golang :(

File: custom-metrics-stackdriver-adapter/pkg/adapter/provider/translator.go [hunk 2]
```diff
@@ -48,6 +48,9 @@ type Translator struct {
 }
 
 // GetSDReqForPods returns Stackdriver request for query for multiple pods.
+// podList is required to be no longer than 100 items. This is enforced by limitation of "one_of()"
```
Comment (piosz) [inline custom-metrics-stackdriver-adapter/pkg/adapter/provider/translator.go]: how about adding such check in the function?

Comment (kawych) [reply custom-metrics-stackdriver-adapter/pkg/adapter/provider/translator.go]: Done.

**Pull request merged**